modules/env_parameters/functions/ENV/fn_ENV_preInit.sqf

```diff
@@ -0,0 +1,53 @@
+#include "script_component.hpp"
+#include "..\..\settings.sqf"
+
+if (isMultiplayer) then {
+	private ["_WeatherParam", "_FogParam", "_overcast", "_wind", "_windDir", "_fog"];
```

Comment:
private <Array> method is much slower than doing private _var method, it's a holdover from before private _var was introduced.

Reply:
Good to know! Fixed.